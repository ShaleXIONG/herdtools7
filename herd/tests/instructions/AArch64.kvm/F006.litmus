AArch64 F006
{
int x=1;
0:X0=x;
[PTE(x)]=(valid:0);
}
  P0          | P0.F           ;
 MOV W1,#2    | MRS X4,ELR_EL1 ;
 LDR W1,[X0]  | ADD X4,X4,#4   ;
 MOV W3,#2    | MSR ELR_EL1,X4 ;
              | ERET           ;

forall 0:X1=2 /\ 0:X3=2
