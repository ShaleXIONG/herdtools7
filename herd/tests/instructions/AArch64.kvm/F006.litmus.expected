Test F006 Required
States 1
0:X1=2; 0:X3=2;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X1=2 /\ 0:X3=2)
Observation F006 Always 1 0
Hash=17b67b7ad6272e5123d8a392ec52856b

